src/main/java/pro/skyjava/course2/examinerservice/service/JavaQuestionService.java

```diff
@@ -0,0 +1,48 @@
+package pro.skyjava.course2.examinerservice.service;
+
+import org.springframework.stereotype.Service;
+import pro.skyjava.course2.examinerservice.domain.Question;
+
+import java.util.*;
+
+@Service
+public class JavaQuestionService implements QuestionService {
+
+    private final Set<Question> questions = new LinkedHashSet<>();
+    private final Random random = new Random();
+
+    @Override
+    public Question add(String question, String answer) {
+        Question q = new Question(question, answer);
+        questions.add(q);
+        return q;
```

Comment:
чтобы не дублировать код нужно было сделать делегацию
```suggestion
        return add(new Question(question, answer));
```